packages/search-ui/src/SearchDriver.js

```diff
@@ -249,7 +252,8 @@ export default class SearchDriver {
     });
 
     this._makeSearchRequest({
-      skipPushToUrl
+      skipPushToUrl,
+      replaceUrl
```

Comment:
Just to make sure I fully understand this functionality: why add a new `replaceUrl` option instead of just making use of the existing `skipPushToUrl` opt? Is it because we do want the URL bar to update?

Reply:
We want the URL to update, but we don't want a new history event to be added.